src/main/webapp/WEB-INF/view/profilepage.jsp

```diff
@@ -0,0 +1,30 @@
+<%@ page import="codeu.model.data.User" %>
+<!DOCTYPE html>
+<html>
+<head>
+ <%@include file="navigationbar.jsp" %>
```

Comment:
Nice! You also want to update the navigationbar.jsp file, so we have a link to get back to this profile page, right?